packages/contracts-bedrock/contracts/test/MerkleTrie.t.sol

```diff
@@ -5,136 +5,118 @@ import { CommonTest } from "./CommonTest.t.sol";
 import { MerkleTrie } from "../libraries/trie/MerkleTrie.sol";
```

Comment:
This file has a large diff because I ran `yarn lint:fix`. I guess that our CI isn't checking formatting of solidity right now.